app/build.gradle

```diff
@@ -93,6 +94,50 @@ android {
     }
 }
 
+tasks.register('codeCoverage', JacocoReport) {
+    dependsOn 'testDebugUnitTest'
+
+    reports {
+        html.required.set(true)
+        xml.required.set(true)
+    }
+
+    classDirectories.setFrom(
```

Comment:
: The compiled class files without instrumentation. Execution data references instructions in these classes, so they're necessary for locating the actual instructions.